corehq/apps/sso/utils/login_helpers.py

```diff
@@ -23,7 +23,14 @@ def process_async_signup_requests(request, user):
             messages.error(
                 request,
                 _("We were unable to create your requested project "
-                  "because the name was already taken."
-                  "Please contact support.")
+                  "because the name was already taken.")
```

Comment:
Was dropping the 'contact support' line here intentional?

Reply:
yes, if you read the commits this was a very intentional choice. Why would we want a user to contact support if the domain name is already taken? they should choose another project name